README.md

```diff
@@ -1,5 +1,41 @@
-A MongoDB Dialect for the Hibernate ORM
+# A MongoDB Dialect for the Hibernate ORM
+
+This project aims to provide a library to seamlessly integrate MongoDB with Hibernate ORM. Hibernate _ORM_ is a powerful **O**bject-**r**elational **m**apping tool. Due to the SQL and JDBC standards, Hibernate ORM could centralize each SQL vendor's idiosyncrasies in the so-called _Hibernate Dialect_. But this project will include a NoSQL member in the Hibernate's Dialect family.
+
+## Overview
+
+MongoDB speaks _MQL_ (**M**ongoDB **Q**uery **L**anguage in JSON format) instead of SQL. This project will create a NoSQL Hibernate Dialect by:
```

Comment:
Let's not use the future tense like this, because then we will have to change it once we've actually done the work  Just say it like we've already done it, using present tense.

Reply:
changed to present tense